api/CSS.json

```diff
@@ -0,0 +1,214 @@
+{
+  "api": {
+    "CSS": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "28"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22",
+            "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+          },
+          "firefox_android": {
+            "version_added": "22",
+            "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+          },
+          "ie": {
+            "version_added": "6"
+          },
+          "opera": {
+            "version_added": "12.1"
+          },
+          "opera_android": {
+            "version_added": "12.1"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "paintWorklet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS/paintWorklet",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "28"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+            },
+            "firefox_android": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+            },
+            "ie": {
+              "version_added": "6"
+            },
+            "opera": {
+              "version_added": "12.1"
+            },
+            "opera_android": {
+              "version_added": "12.1"
```

Comment:
`52`

Reply:
It's older than 52, but I can't say exactly how old.

Reply:
My comment was originally on line 104, but GitHub moved weirdly.